pkg/util/tags_builder.go

```diff
@@ -86,16 +92,25 @@ func (tb *TagsBuilder) Truncate(len int) {
 
 // Get returns the internal slice
 func (tb *TagsBuilder) Get() []string {
+	if tb == nil {
+		return nil
+	}
```

Comment:
I think it's unusual for methods to support a nil receiver -- perhaps a doc comment on the struct or each method would be useful?